data/campaigns/Eastern_Invasion/scenarios/04a_An_Elven_Interlude.cfg

```diff
@@ -411,10 +414,12 @@
         {MOVE_UNIT id=$second_unit.id 15 1}
         [message]
             speaker=$second_unit.id
+            #po: capturing Addogin, spoken by the unit fighting him
             message= _ "Oh no you don’t! You and your band of mercenaries have caused us enough trouble! How do we know you won’t rally your men and attack us again?"
         [/message]
         [message]
             speaker=Gweddry
+            #po: capturing Addogin
```

Comment:
Is this really sufficient to get across the fact that "him" in the text refers to Addogin?

Reply:
What would you suggest instead?

With "tie him up" in the string, I think "capturing X" is clear enough.